setup.py

```diff
@@ -19,7 +19,7 @@
 setup(name='bruker',
       version=__version__,
       description='Bruker PvDataset Loader',
-      python_requires='>3.5, <3.8',
+      python_requires='>3.5, <3.9',
```

Comment:
if you require >3.6, you could convert all your string formatting to f-strings

Reply:
Might consider your suggested change (make it >3.6) to put items for the next version. Thank you for the suggestion!